Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-widgets/index.jsx

```diff
@@ -4,22 +4,26 @@ import { cloudinary } from '../../../config/default'
 
 const RenderCloudinaryGalleryWidget = ({ cloudinaryImageGallery = {} }) => {
 
-    useEffect(() => {
-        if (cloudinaryImageGallery && cloudinaryImageGallery.galleryEnabled) {
-            const gallery = document.querySelector('#cld-gallery')
-            const galleryOptions = cloudinaryImageGallery.cloudinaryImage.galleryWidget.options
-            galleryOptions.queryParam = cloudinary.CLD_PGW_TRACKING_PARAM
-            if (typeof window !== 'undefined' && window.cloudinary && window.cloudinary.galleryWidget) {
-                if (window.cldGallery) {
-                    gallery.style.visibility = 'hidden'
-                    cldGallery.update(galleryOptions)
-                    gallery.style.visibility = 'visible'
-                } else {
-                    window.cldGallery = window.cloudinary.galleryWidget(galleryOptions)
-                }
+    const renderGallery = async () => {
+        const gallery = document.querySelector('#cld-gallery')
+        const galleryOptions = cloudinaryImageGallery.cloudinaryImage.galleryWidget.options
+        galleryOptions.queryParam = cloudinary.CLD_PGW_TRACKING_PARAM
+        if (typeof window !== 'undefined' && window.cloudinary && window.cloudinary.galleryWidget) {
+            if (window.cldGallery) {
+                gallery.style.visibility = 'hidden'
+                await cldGallery.update(galleryOptions)
+                gallery.style.visibility = 'visible'
+            } else {
+                window.cldGallery = window.cloudinary.galleryWidget(galleryOptions)
                 cldGallery.render()
```

Comment:
for consistency, I would use window.cldGallery

Reply:
apologies for this miss, I just updated